app/src/main/java/com/google/droidjump/GameView.java

```diff
@@ -173,27 +204,46 @@ private void sleep() {
     private void drawScene() {
         if (surfaceHolder.getSurface().isValid()) {
             Canvas canvas = getHolder().lockCanvas();
+            // Cleaning previous canvas
             canvas.drawColor(Color.WHITE);
             String levelHeader = String.format("%s %d", GAME_LEVEL_HEADER, currentLevel);
             float levelPaintY = screenMargin + levelPaint.getTextSize();
             canvas.drawText(/* text= */ levelHeader, /* x= */ screenMargin, levelPaintY, levelPaint);
             drawDroid(canvas);
+            drawObstacles(canvas);
+            drawPlatform(canvas);
+
+            // Drawing canvas with all elements
             surfaceHolder.unlockCanvasAndPost(canvas);
         }
     }
 
     private void failGame() {
+        isPlaying = false;
         findNavController(this).navigate(R.id.action_game_screen_to_game_failure_screen);
     }
 
     private void winGame() {
+        isPlaying = false;
         findNavController(this).navigate(R.id.action_game_screen_to_game_success_screen);
     }
 
     private void drawDroid(Canvas canvas) {
         canvas.drawBitmap(droid.getBitmap(), droid.getX(), droid.getY(), /* paint= */ null);
     }
 
+    private void drawPlatform(Canvas canvas) {
+        int platformY = screenY - platform.getHeight();
+        for (int curPlatformX = platformX; curPlatformX < screenX; curPlatformX += platform.getWidth())
```

Comment:
nit: add {} so this would be:
        for (int curPlatformX = platformX; curPlatformX < screenX; curPlatformX += platform.getWidth())
{            canvas.drawBitmap(platform, curPlatformX, platformY, /* paint= */ null);
}